mod.ts

```diff
@@ -132,10 +136,14 @@ export interface BuildOptions {
 
 /** Builds the specified Deno module to an npm package using the TypeScript compiler. */
 export async function build(options: BuildOptions): Promise<void> {
+  if (options.scriptModule == false && options.esModule == false) {
```

Comment:
Would you be able to add one test for this that just verifies this throws? That way we can ensure it doesn't break in the future.

Reply:
Alright!